### ISSUE TITLE
Expose metrics collected while running background sweep

### ISSUE THREAD

**Issue by rhero** [palantir/atlasdb#1537]
We're planning on enabling the background sweep job by default (#1118) and so we should instrument some metrics to be collected with other dropwizard metrics (#1429). The metrics we expose should inform decisions on how to optimize AtlasDB clients and detect performance issues before they become P0s. 

Some uestions I'd want to be able to answer are:

1. Which tables are we sweeping the most (in terms of total disk space and number of cells).
2. Is this table being swept more _now_ than it was _last release_? (probably just same metrics as above and monitor change)
3. How much resources am I consuming with sweep?

@tpetracca @clockfort @schlosna @gsheasby @jboreiko in case you have thoughts on what we could add.

**Comment by tpetracca** [comment]
We should also expose all the things we log (and maybe in more ingestible useful ways)
- deleted cells per table
- unique cells per table
- deleted cells per row for a given table
  - median (generally how dirty the table is)
  - max (if significantly different from median, may imply special bad row keys)

**Comment by rhero** [comment]
Would like to add these in parallel to rolling out background sweep to internal services. Not the most urgent, but would like to get to soon.

**Comment by harrybiddle** [comment]
I suggest this is pretty important for us to track how the background sweep rollout is going.

If we use the metrics collector, we'll lose any metrics every time the service restarts. We might want to just serve up the metrics that are already saved down in the KVS.

**Comment by ashrayjain** [comment]
@tpetracca @clockfort do you guys think we could use the `TombstoneScannedHistogram` metric here? It's not really preemptive though and only gives us an estimate on how many tombstones Cassandra has encountered per table.

**Comment by gsheasby** [comment]
I'd like to add more metric in the fullness of time - for now only basic metrics have been added. Less urgent than other stuff for now, so going to ping it back to 'new issues' so we can discuss it at the next triage session.

**Comment by rhero** [comment]
Closing out because we want to see an end to end use case for the first set of metrics before exposing more. Will file a separate ticket when we have specific use cases for new metrics.
